### PR TITLE
Feature/alpaykuru nuc224 transfer scroll view component

### PR DESCRIPTION
**Pull request by alpaykuru**
Screenshot from 2019-05-02 11-26-52

### CODE REVIEW

File: docs/.vuepress/components/ScrollViewDoc.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<template>
+<StackLayout class="scroll-view" orientation="horizontal">
+        <ScrollView :orientation="scrollType" id="scrollview1" class="scroll-view__vertical" :scrollBarIndicatorVisible="scrollBarIndicatorVisible" >
+          <StackLayout orientation="vertical">
+            <div>Native Vue Web project is web components for Nativescript-Vue. NativeScript is an open source framework for building truly native mobile applications using JavaScript.</div>
```
Comment (ahmetceylan) [inline docs/.vuepress/components/ScrollViewDoc.vue]: you can use Label component to print text

File: docs/.vuepress/components/ScrollViewDoc.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          </StackLayout>
+        </ScrollView>
+       <StackLayout orientation="horizontal">
+            <Label text="Scroll Bar Enabled:" id="scrollview_enabled_label" class="scroll-view__margin_left" width="200"></Label>
+            <input type="checkbox" v-model="scrollBarIndicatorVisible" id="scrollview_horizontal_input" />
```
Comment (ahmetceylan) [inline docs/.vuepress/components/ScrollViewDoc.vue]: please use textfield instead of input, replace all inputs with textfield

File: docs/.vuepress/components/ScrollViewDoc.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
+<template>
+<StackLayout class="scroll-view" orientation="horizontal">
+        <ScrollView :orientation="scrollType" id="scrollview1" class="scroll-view__vertical" :scrollBarIndicatorVisible="scrollBarIndicatorVisible" >
```
Comment (ahmetceylan) [inline docs/.vuepress/components/ScrollViewDoc.vue]: please update id of the scrollview with something like "scrollview_container"

File: docs/.vuepress/components/ScrollViewDoc.vue (hunk 1 above)
Comment (ahmetceylan) [reply docs/.vuepress/components/ScrollViewDoc.vue]: and please replace "nativescript vue web" references with vuexp

File: docs/.vuepress/config.js [hunk 4]
```diff
@@ -37,8 +37,8 @@ module.exports = {
               "components/search-bar.md",
               "components/button.md",
               "components/textfield.md",
+              "components/scrollview.md",
               "components/slider.md",
-              "components/textview.md",
```
Comment (ahmetceylan) [inline docs/.vuepress/config.js]: why did you delete textview ?

File: src/main.js [hunk 5]
```diff
@@ -1,8 +1,13 @@
+const placeholder = {};
+
+export default placeholder;
+export { default as Label } from './components/Label';
```
Comment (ahmetceylan) [inline src/main.js]: No need to import/export components from main.js
we should import and use them directly in the components when we need

File: docs/.vuepress/config.js (hunk 4 above)
Comment (alpaykuru) [reply docs/.vuepress/config.js]: Because of conflict.After resolve conflict there was 2 textfield component and i deleted

File: tests/unit/layouts/AbsoluteLayout.spec.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import { expect } from 'chai';
 import { mount } from '@vue/test-utils';
-import { Label } from '../../../src/main';
+import Label from '../../../src/components/Label';
 import AbsoluteLayout from '../../../src/layouts/AbsoluteLayout.vue';
```
Comment (volkanakbayir) [inline tests/unit/layouts/AbsoluteLayout.spec.js]: can you emit the .vue extension from imports for standardisation?

File: docs/.vuepress/components/ScrollViewDoc.vue [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+      scrollBarIndicatorVisible:false,
+      horizontal:false,
+      scrollText: ''
+    };
+  },
+  methods: {
+    onScroll(){
+      this.scrollText = 'scrolled';
+    },
+    clearScrollText() {
```
Comment (Serhatkemal) [inline docs/.vuepress/components/ScrollViewDoc.vue]: This method is never used. Can you remove it?
